### ISSUE TITLE
Password reveal button isn't shown

### ISSUE THREAD

**Issue by vadd98** [AvaloniaUI/Avalonia#5665]
**Describe the bug**

Password reveal button isn't shown in TextBox when I use the PasswordChar property. I found in the TextBox template that there is a reveal button, but I can't found how I can make it appear.

**To Reproduce**

1. Add a TextBox with PasswordChar
2. Start typing in the TextBox

**Expected behavior**

Password reveal button has to appear when typing

**Screenshots**

**Desktop (please complete the following information):**

- OS: [Windows 10, MacOS Big Sur]
- Version [0.10.0]

**Additional context**

**Comment by luthfiampas** [comment]
Have you already add `revealPasswordButton` class?

**Comment by maxkatz6** [comment]
<TextBox Classes="revealPasswordButton" />

**Comment by vadd98** [comment]
> Have you already add `revealPasswordButton` to the `TextBox.Classes`?

Yes I did, but now I think I'm doing something wrong. I tried to put multiple styles to my TextBox like this:

`<TextBox x:Name="PasswordField" Classes="LoginTextBox.revealPasswordButton" PasswordChar="*"/>`

but I think it is wrong. Indeed, if I only put `revealPasswordButton` it works. 
What can I do? I have to copy the `revealPasswordButton`style in my custom style and use just one class or is there any better solutions?
